<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>575</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MySFMLCanvas" name="MyCanvas" native="true"/>
    </item>
    <item>
     <widget class="QGroupBox" name="InfoBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Info</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QGroupBox" name="circuit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>32</y>
         <width>226</width>
         <height>150</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>Circuit</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="circuitName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="circuitName_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="circuitDieArea">
          <property name="text">
           <string>Die Area</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="circuitDieArea_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="circuitCells">
          <property name="text">
           <string>Cells</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="circuitCells_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="circuitPins">
          <property name="text">
           <string>Pins</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="circuitPins_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="circuitNets">
          <property name="text">
           <string>Nets</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="circuitNets_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="selectedCell">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>188</y>
         <width>226</width>
         <height>242</height>
        </rect>
       </property>
       <property name="title">
        <string>Selected Cell</string>
       </property>
       <widget class="QWidget" name="NameType" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>17</y>
          <width>61</width>
          <height>196</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="selectedCellName">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="selectedCellType">
           <property name="text">
            <string>Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="selectedCellSize">
           <property name="text">
            <string>Size  w:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="selectedCell_h">
           <property name="text">
            <string>         h:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="selectedCellPosition">
           <property name="text">
            <string>Pos.  x:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="selectedCell_y">
           <property name="text">
            <string>         y:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="NameType_2" native="true">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>17</y>
          <width>175</width>
          <height>230</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="selectedCellName_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="selectedCellType_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="selectedCell_w_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="selectedCell_h_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="selectedCell_x_2">
           <property name="minimum">
            <double>-10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="selectedCell_y_2">
           <property name="minimum">
            <double>-10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10000000000000000564754110205208414148406263819830583747005651641554565639675781971892197615894599829797681693475363620965659806446069238773051601456032797794197839403040623198185642380825912769195995883053017532724018486962951290880.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectedCellButton">
           <property name="text">
            <string>Update Position</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="find">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>437</y>
         <width>224</width>
         <height>88</height>
        </rect>
       </property>
       <property name="title">
        <string>Find cell by name</string>
       </property>
       <widget class="QLineEdit" name="findName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="findNameButton">
        <property name="geometry">
         <rect>
          <x>176</x>
          <y>30</y>
          <width>41</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Go</string>
        </property>
       </widget>
       <widget class="QLabel" name="findErro">
        <property name="geometry">
         <rect>
          <x>13</x>
          <y>56</y>
          <width>156</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="findNameButtonClear">
        <property name="geometry">
         <rect>
          <x>176</x>
          <y>57</y>
          <width>41</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_LEF_DEF">
     <property name="title">
      <string>Open LEF/DEF</string>
     </property>
     <addaction name="actionICCAD_2017"/>
     <addaction name="actionICCAD_2015"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionPNG"/>
     <addaction name="actionSVG"/>
    </widget>
    <addaction name="menuOpen_LEF_DEF"/>
    <addaction name="menuExport"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionSlot_1"/>
    <addaction name="actionSlot_2"/>
    <addaction name="actionSlot_3"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionICCAD_2017">
   <property name="text">
    <string>ICCAD 2017</string>
   </property>
  </action>
  <action name="actionICCAD_2015">
   <property name="text">
    <string>ICCAD 2015</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSlot_1">
   <property name="text">
    <string>Slot 1</string>
   </property>
  </action>
  <action name="actionSlot_2">
   <property name="text">
    <string>Slot 2</string>
   </property>
  </action>
  <action name="actionSlot_3">
   <property name="text">
    <string>Slot 3</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionSVG">
   <property name="text">
    <string>SVG</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MySFMLCanvas</class>
   <extends>QWidget</extends>
   <header>view/mysfmlcanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
